packages/transition-frontend/src/services/definitions/DefinitionsService.ts

```diff
@@ -5,23 +5,49 @@
  * License text available at https://opensource.org/licenses/MIT
  */
 
-import React from 'react';
 import serviceLocator from 'chaire-lib-common/lib/utils/ServiceLocator';
 import * as Status from 'chaire-lib-common/lib/utils/Status';
 import { Dictionary } from 'lodash';
 
+const definitionCache: { [label: string]: Dictionary<string> | 'loading' | 'error' } = {};
+
 export const getDefinitionFromServer = (
```

Comment:
Maybe just add a comment that explains the definition will only be gotten from the server the first time and after that is taken from the cache?